README.md

```diff
@@ -276,6 +276,13 @@ These instrumentations are hosted at <https://github.com/open-telemetry/opentele
 
 ## Upgrade guidelines
 
+### 0.28.x to 0.29.x
+
+- `@opentelemetry/exporter-trace-otlp-http` is now exporting `scopeSpans` instead of `instrumentationLibrarySpans`
+  - this exporter now requires collector version `0.48` and up.
```

Comment:
does this mean the new version of the exporter will not work with older versions of the collector?
whats the recourse if we need to talk to the older collector?

Reply:
Yes. I found that at least for the `http/json` exporter this is the case when following the recommendation of the spec.

It is recommended to set both keys, however the old collector receiver drops requests with unexpected keys.

The other exporters (`otlp/proto` and `otlp/grpc`) will still work, though.

Reply:
I see thank you

Reply:
I believe the collector accepts both equally if I am not mistaken in order to ensure maximum compatibility.

Reply:
> I believe the collector accepts both equally if I am not mistaken in order to ensure maximum compatibility.

Ah oops I realized my mistake that would be compatibility in the wrong direction. We would need to emit both to be used with old collectors. The spec states that this is an optional feature we could implement if we find users have a hard time updating the collector.

Reply:
> We would need to emit both to be used with old collectors. The spec states that this is an optional feature we could implement if we find users have a hard time updating the collector.

@dyladan  [Looks like the collector itself has a bug in this functionality](https://github.com/open-telemetry/opentelemetry-js/pull/2929#discussion_r860846247). I think there is not much we can do to support new exporters and old collectors 

@pichlermarc will open an issue for the collector to fix this behavior

Reply:
Yes. It won't be possible to fix this behavior retroactively, I suppose, but it can be prevented in the future. https://github.com/open-telemetry/opentelemetry-collector/issues/5312